spyder/app/tests/test_mainwindow.py

```diff
@@ -672,6 +672,29 @@ def test_open_files_in_new_editor_window(main_window, qtbot):
 
 
 @flaky(max_runs=3)
+def test_close_when_file_is_changed(main_window, qtbot):
```

Comment:
This test is hanging in Travis with the pip wheels. So please skip it when `PYQT_WHEEL`